### PR TITLE
bash completion

### PR DESCRIPTION
**Pull request by 1ace**
Similar to the existing shell completion for zsh, this adds support for completion in bash

### CODE REVIEW

File: completions/bash/swaygrab [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# swaygrab(1) completion
```
Comment (emersion) [inline completions/bash/swaygrab]: `swaygrab` has been removed. We should remove completions too.

Comment (1ace) [reply completions/bash/swaygrab]: OK, sure.
I haven't followed the development of sway; why was `swaygrab` removed? Is its functionality available some other way?

Comment (emersion) [reply completions/bash/swaygrab]: Yeah, you can now use tools like [grim](https://github.com/emersion/grim).

Comment (emersion) [reply completions/bash/swaygrab]: If possible, can you open a PR to remove all `swaygrab` references in docs and completion?

```
meson.build
185:		'completions/zsh/_swaygrab',

sway/sway.1.scd
95:*sway*(5) *swaymsg*(1) *swaygrab*(1) *sway-input*(5) *sway-bar*(5)

swayidle/swayidle.1.scd
61:*sway*(5) *swaymsg*(1) *swaygrab*(1) *sway-input*(5) *sway-bar*(5)

completions/zsh/_swaygrab
1:#compdef swaygrab
6:# Completion script for swaygrab in sway wm (http://swaywm.org)
```

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**